bin/use_simulator.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+
+set -o pipefail
+
+project="$1"
+scheme="$2"
+
+xcodebuild -project "$project" -scheme "$scheme" -showBuildSettings \
+  | grep CORRESPONDING_SIMULATOR_SDK_NAME \
+  2>&1 > /dev/null
```

Comment:
I'm essentially trying to ignore any of the output. All I care about is the `grep` return status. Was there or wasn't there a key called `CORRESPONDING_SIMULATOR_SDK_NAME`.